src/Mongo2Go/Helper/MongodArguments.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Mongo2Go.Helper
+{
+    public static class MongodArguments
+    {
+        private const string ArgumentSeparator = "--";
+        private const string Space = " ";
+
+        /// <summary>
+        /// This method will cleanup the additional mongod arguments to make sure the Mongo2Go arguments are not overriden
+        /// </summary>
```

Comment:
> This method will cleanup the additional mongod arguments to make sure the Mongo2Go arguments are not overriden

This could confuse people. Either we should inform the user somehow (e.g. with one exception) or we should overwrite the original Mongo2Go values.  This could be a useful feature. But it's your feature, so it's up to you how it should work.
In any case we need some documentation in the README.md

Reply:
Thanks for the comment.
In regards to overriding the original Mongo2Go values, due to my lack of knowledge on how Mongo2Go expects mongod to work, I don't think that should be allowed. So, I'll take the exception route... and I'll add documentation in the README.md